enterprise/coderd/coderd.go

```diff
@@ -884,9 +883,9 @@ func (api *API) updateEntitlements(ctx context.Context) error {
 		if initial, changed, enabled := featureChanged(codersdk.FeatureWorkspacePrebuilds); shouldUpdate(initial, changed, enabled) || api.PrebuildsReconciler == nil {
 			reconciler, claimer, metrics := api.setupPrebuilds(enabled)
 			if api.PrebuildsReconciler != nil {
-				stopCtx, giveUp := context.WithTimeoutCause(context.Background(), time.Second*30, errors.New("gave up waiting for reconciler to stop"))
+				stopCtx, giveUp := context.WithTimeoutCause(context.Background(), time.Second*30, xerrors.New("gave up waiting for reconciler to stop"))
```

Comment:
Also out of scope of this PR: stopping the prebuilds reconciler on-the-fly due to a licensing change is a pretty big deal. I assume it'll have to scale down all of the pools and delete all of the resources. This will be a pretty expensive operation and potentially disruptive in the case of an operator accidentally forgetting to insert an updated license.

Reply:
> I assume it'll have to scale down all of the pools and delete all of the resources.

That won't be the case. Stopping the reconciler just stops the reconciliation loop, and does not affect any of the running prebuilds.

We _could_ implement that, but the likelihood of someone removing their premium license is low. We'll punt for now. What we've implemented so far is just the ability to handle license changes (mostly for tests).